dev-python/python-poppler-qt5/python-poppler-qt5-0.75.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+inherit distutils-r1 flag-o-matic qmake-utils
+
+DESCRIPTION="A python binding for libpoppler-qt5"
+HOMEPAGE="https://github.com/wbsoft/python-poppler-qt5"
+SRC_URI="https://github.com/wbsoft/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	app-text/poppler[qt5]
+	dev-python/PyQt5[${PYTHON_USEDEP}]
+	>=dev-python/sip-4.19:=[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+
+python_configure_all() {
+	append-cxxflags -std=c++11
```

Comment:
Please try without, I don't think it should be necessary at this point.

Reply:
Right you are. That seems to make the entire python_configure_all() redundant, the defaults work just fine.

Reply:
Wait, does that mean I can ditch flag-o-matic and qmake-utils eclasses too? Testing.

Reply:
exactly, and I don't see qmake-utils.eclass in use anywhere anyway.